package.json

```diff
@@ -22,21 +22,23 @@
   },
   "homepage": "https://github.com/appear-sh/introspector-js#readme",
   "devDependencies": {
+    "@types/aws-lambda": "^8.10.140",
     "@types/express": "^4.17.21",
-    "@types/lodash": "^4.17.0",
-    "@types/node": "^20.11.28",
+    "@types/lodash": "^4.17.6",
+    "@types/node": "^20.14.9",
     "@types/xxhashjs": "^0.2.4",
-    "axios": "^1.6.8",
+    "axios": "^1.7.2",
     "body-parser": "^1.20.2",
-    "express": "^4.18.3",
-    "msw": "^2.2.4",
-    "prettier": "^3.2.5",
-    "typescript": "^5.4.2",
-    "vitest": "^1.4.0"
+    "express": "^4.19.2",
+    "msw": "^2.3.1",
+    "prettier": "^3.3.2",
+    "typescript": "^5.5.2",
+    "vitest": "^1.6.0"
   },
   "dependencies": {
-    "@mswjs/interceptors": "^0.26.8",
-    "browser-or-node": "3.0.0-pre.0",
+    "@mswjs/interceptors": "^0.30.0",
+    "@vercel/functions": "^1.0.2",
```

Comment:
Hmm. Should this be a peer dep? Pulling this in feels strange. Will using `waitUntil` just noop in a non-vercel environment? Or will it throw?

Reply:
making it peer dep would increase number of steps consumer have to do. And yea in non-vercel env it is noop and is working just fine. I've tested it locally and here's [the code](https://github.com/vercel/vercel/blob/main/packages/functions/index.js#L15) which doesn't do anything if `waitUntil` isn't present in `globalThis`.